chat-plugins/room-events.js

```diff
@@ -94,11 +94,53 @@ exports.commands = {
 		help: function (target, room, user) {
 			return this.parse('/help roomevents');
 		},
+		sortby: function (target, room, user) {
+			if (!room.chatRoomData) return this.errorReply("This command is unavailable in temporary rooms.");
+			if (!room.events || !Object.keys(room.events).length) {
+				return this.errorReply("There are currently no planned upcoming events for this room.");
+			}
+			if (!this.can('ban', null, room)) return false;
+			let multiplier = 1;
+			let columnName = "";
+			let order = "";
```

Comment:
no need to declare this outside of the block it's used in

Reply:
moved order to the conditional block.